### DISCUSSION TITLE
Поле "Число", редактирование только при вводе

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#123] [Как это сделать?] [answered]
Здравствуйте. 
Подскажите плиз с такой задачкой:
Простая таблица. Столбец "Число". Как сделать что бы это число можно было вводить вручную только при добавлении строки, когда строка добавлено - изменять значение может только код.
Спасибо

**Comment by erpic-ru** [comment, marked as answer]
Добрый день.

`Изменяемое при добавлении` — `включено`

`Доступ на изменение` — `выключено`

Вот раздел учебного курса: https://ru.docs.totum.online/adding-rows-and-accesses-to-add-and-change

Например, блокировку можно сделать через форматирование:

`f1=:setFormat(condition: #id!=""; block: true)`

Но все блокировки через формитирование невзломостойкие!

**Reply by ttm-alexeypolunin** [reply]
Дополнил 👆
